packages/schema-dot-org-types/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "schema-dot-org-types",
+  "version": "0.1.0",
+  "description": "Classes with flow types to encapsulate schema.org markup attributes.",
+  "main": "index.js",
```

Comment:
Generated files should live in the generated directory, `lib`, not as siblings of versioned files. Nobody will be importing `schema-dot-org-types/lib/index`, as it is nonsensical.